### PR TITLE
feat(Workspaces): add workspace invitations list to django admin

### PR DESCRIPTION
**Pull request by cheikhgwane**
Setup workspace invitations on django admin view.

## Changes

Please list / describe the changes in the codebase for the reviewer(s).

- Register WorkspaceInvitation to django admin. 

## How/what to test

A few words about what needs to be tested, along with specific testing instructions if needed.

## Screenshots / screencast

image

### CODE REVIEW

File: hexa/workspaces/admin.py [hunk 1]
```diff
@@ -42,3 +48,16 @@ class ConnectionFieldAdmin(admin.ModelAdmin):
         "connection",
         "code",
     )
+
+
+@admin.register(WorkspaceInvitation)
+class WorkspaceInvitationAdmin(admin.ModelAdmin):
+    list_display = (
+        "id",
```
Comment (pvanliefland) [inline hexa/workspaces/admin.py:56]: We can probably skip the `id` and keep `email` as the first column

**Review by pvanliefland — CHANGES_REQUESTED** [review]
See comment about `id` - otherwise, looking good

File: hexa/workspaces/admin.py (hunk 1 above)
Comment (cheikhgwane) [reply hexa/workspaces/admin.py:56]: Okay.

**Review by pvanliefland — APPROVED** [review]
👍

**Pull request merged**